client/src/app/payroll/payroll-list/payroll-list.component.ts

```diff
@@ -1,16 +1,16 @@
 import { Component } from '@angular/core';
-import { Payroll } from 'src/app/data/state/payroll-model';
-import { faSmileBeam, faSkull } from '@fortawesome/free-solid-svg-icons';
+import { faSkull, faSmileBeam } from '@fortawesome/free-solid-svg-icons';
 import { DataService } from 'src/app/data/data-service';
+import { Payroll } from 'src/app/data/state/payroll-model';
 
 @Component({
   selector: 'app-payroll-list',
   templateUrl: './payroll-list.component.html',
   styleUrls: ['./payroll-list.component.scss'],
 })
 export class PayrollListComponent {
-  faSkull = faSkull;
-  faSmileBeam = faSmileBeam;
+  readonly faSkull = faSkull;
```

Comment:
👍 this is a good pattern that I like to use. `readonly` for everything that doesn't change.